doc/source/reference/arrays.classes.rst

```diff
@@ -39,7 +39,7 @@ Special attributes and methods
 
 NumPy provides several hooks that classes can customize:
 
-.. method:: class.__numpy_ufunc__(ufunc, method, i, inputs, **kwargs)
+.. method:: class.__array_ufunc__(ufunc, method, inputs, **kwargs)
 
    .. versionadded:: 1.11
```

Comment:
Oh, https://github.com/charris/numpy/pull/6/files#diff-38494a4fd388b5a497f4082aabd236e7 switches it to 1.13 though.